static/app/views/issueDetails/streamline/metricIssueChart.tsx

```diff
@@ -0,0 +1,116 @@
+import {useApiQuery} from 'sentry/utils/queryClient';
+import useApi from 'sentry/utils/useApi';
+import useOrganization from 'sentry/utils/useOrganization';
+import type {TimePeriodType} from 'sentry/views/alerts/rules/metric/details/constants';
+import {
+  getFilter,
+  getPeriodInterval,
+} from 'sentry/views/alerts/rules/metric/details/utils';
+import {
+  Dataset,
+  type MetricRule,
+  TimePeriod,
+} from 'sentry/views/alerts/rules/metric/types';
+import {extractEventTypeFilterFromRule} from 'sentry/views/alerts/rules/metric/utils/getEventTypeFilter';
+import {isCrashFreeAlert} from 'sentry/views/alerts/rules/metric/utils/isCrashFreeAlert';
+import {isCustomMetricAlert} from 'sentry/views/alerts/rules/metric/utils/isCustomMetricAlert';
+
+const MetricChart = lazy(
+  () => import('sentry/views/alerts/rules/metric/details/metricChart')
+);
+
+export function MetricIssueChart({
+  event,
+  group,
+  project,
+}: {
+  event: Event | undefined;
+  group: Group;
+  project: Project;
+}) {
+  const api = useApi();
+  const organization = useOrganization();
+
+  const ruleId = event?.contexts?.metric_alert?.alert_rule_id;
+  const {data: rule} = useApiQuery<MetricRule>(
+    [
+      `/organizations/${organization.slug}/alert-rules/${ruleId}/`,
+      {
+        query: {
+          expand: 'latestIncident',
+        },
+      },
+    ],
+    {
+      staleTime: Infinity,
+      retry: false,
+    }
+  );
+
+  const openPeriod = group.openPeriods?.[0];
+  const timePeriod = useMemo((): TimePeriodType | null => {
+    if (!openPeriod) {
+      return null;
+    }
+    const start = openPeriod.start;
+    let end = openPeriod.end;
+    if (!end) {
+      end = new Date().toISOString();
+    }
+    return {
+      start,
+      end,
+      period: TimePeriod.SEVEN_DAYS,
+      usingPeriod: false,
+      label: t('Custom time'),
+      display: (
+        <Fragment>
+          <DateTime date={moment.utc(start)} />
+          {' — '}
+          <DateTime date={moment.utc(end)} />
+        </Fragment>
+      ),
+      custom: true,
+    };
+  }, [openPeriod]);
+
+  if (!rule || !timePeriod) {
+    return null;
+  }
+
+  const {dataset, aggregate, query} = rule;
+  let formattedAggregate = aggregate;
+  if (isCustomMetricAlert(aggregate)) {
+    formattedAggregate = formatMRIField(aggregate);
+  }
+  const eventType = extractEventTypeFilterFromRule(rule);
+  const queryWithTypeFilter =
+    dataset === Dataset.EVENTS_ANALYTICS_PLATFORM
+      ? query
+      : (query ? `(${query}) AND (${eventType})` : eventType).trim();
+
+  return (
+    <LazyLoad
+      LazyComponent={MetricChart}
+      api={api}
+      rule={rule}
+      timePeriod={timePeriod}
+      organization={organization}
+      project={project}
+      interval={getPeriodInterval(timePeriod, rule)}
+      query={isCrashFreeAlert(dataset) ? query : queryWithTypeFilter}
+      filter={getFilter(rule)}
+      formattedAggregate={formattedAggregate}
+    />
```

Comment:
I think this is solid for now. We'll eventually want to just pull the data so we can display it with the new design and have it match other issue types, but for now this works great.